src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutPopupPanel.cs

```diff
@@ -35,6 +40,32 @@ public FlyoutBasePopupPanel(FlyoutBase flyout) : base(flyout._popup)
 		internal override FlyoutBase Flyout => _flyout;
 
 		protected override int PopupPlacementTargetMargin => 5;
+
+		private protected override void OnPointerPressedDismissed(PointerRoutedEventArgs args)
+		{
+			if (Flyout.OverlayInputPassThroughElement is not UIElement passThroughElement
+				|| !Flyout.EnumerateAncestors().Contains(passThroughElement))
```

Comment:
Wouldn't this cause enumeration of ancestors when passthrough is not set at all ?

Reply:
It should not.

Reply:
It should not, the first condition will return `true` so the second will not be evaluated.